### PR TITLE
Enhacement bond parser

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by xiangce — APPROVED** [review]

File: insights/parsers/bond.py [hunk 1]
```diff
@@ -113,6 +121,14 @@ def parse_content(self, content):
...
             elif line.strip().startswith("Currently Active Slave"):
                 self._active_slave = line.split(":", 1)[1].split()[0]
+            elif line.strip().startswith("MII Status: "):
+                self._mii_status.append(line.strip().split(':', 1)[1].strip())
+            elif line.strip().startswith("Link Failure Count: "):
+                self._slave_link_failure_count.append(line.strip().split(':', 1)[1].strip())
+            elif line.strip().startswith("Speed: "):
+                self._slave_speed.append(line.strip().split(':', 1)[1].strip())
+            elif line.strip().startswith("Duplex: "):
+                self._slave_duplex.append(line.strip().split(':', 1)[1].strip())
```
Comment (bfahr) [inline insights/parsers/bond.py]: @rasrivas-redhat there's duplication in this code, why not make it a dict based on a list of keywords that you are looking for?  That would simplify this code.

Comment (rasrivas-redhat) [reply insights/parsers/bond.py]: @bfahr - updated the doc string as per our discussion, please review it.

Comment (bfahr) [reply insights/parsers/bond.py]: After discussing with @rasrivas-redhat I believe we should go ahead with his existing implementation.

**Comment by rasrivas-redhat** [comment]
@bfahr - if you are good with PR, can you please merge this PR?

**Review by xiangce — APPROVED** [review]

**Pull request merged**